api/scpca_portal/models/project.py

```diff
@@ -1159,3 +1159,13 @@ def update_downloadable_sample_count(self):
         self.downloadable_sample_count = downloadable_sample_count
         self.unavailable_samples_count = unavailable_samples_count
         self.save()
+
+    def get_filtered_data_file_paths(self, download_config: Dict) -> List[Path]:
```

Comment:
We support merged data in more than one format so we still need to filter on that here as well.